src/pages/resultsView/SurvivalStoreHelper.ts

```diff
@@ -50,3 +49,47 @@ export function getPatientSurvivals(
         return [];
     }
 }
+
+export function getClinicalDataOfPatientSurvivalStatus(
+    survivalClinicalDataGroupByUniquePatientKey: {
+        [patientKey: string]: ClinicalData[];
+    },
+    targetUniquePatientKeys: string[],
+    statusAttributeId: string,
+    monthsAttributeId: string
+): ClinicalData[] {
+    if (targetUniquePatientKeys) {
+        return targetUniquePatientKeys.reduce(
+            (
+                patientSurvivalStatusData: ClinicalData[],
+                uniquePatientKey: string
+            ) => {
+                const clinicalData =
+                    survivalClinicalDataGroupByUniquePatientKey[
+                        uniquePatientKey
+                    ];
+                if (clinicalData) {
+                    const statusClinicalData = clinicalData.find(
+                        c => c.clinicalAttributeId === statusAttributeId
+                    );
+                    const monthsClinicalData = clinicalData.find(
+                        c => c.clinicalAttributeId === monthsAttributeId
+                    );
+                    if (
+                        statusClinicalData &&
+                        monthsClinicalData &&
+                        statusClinicalData.value !== 'NA' &&
```

Comment:
gauranteed to be NA?  i mean could it be lowercase?

Reply:
either way lets make it a constant and put it somewhere, maybe even in the /api/ folder in an api constants file

Reply:
@alisman @adamabeshouse No, it's not guaranteed to be NA. I should make a constant in SurvivalUtil. The validator will allow the following values (ignore case and ignore whitespaces): `"[not applicable]", "[not available]", "[pending]", "[discrepancy]", "[completed]", "[null]", "", "na"`

https://github.com/cBioPortal/cbioportal/blob/master/core/src/main/scripts/importer/validateData.py#L2156
https://github.com/cBioPortal/cbioportal/blob/master/core/src/main/scripts/importer/validateData.py#L2468-L2470

Reply:
@alisman @adamabeshouse Added a constant and a function in `SurvivalUtil` to check if a value is null. Can I get your review again? Thanks!